3761/3/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,27 @@
+require 'rack-flash'
+require 'dotenv'
+require_relative '../helpers/user_helper'
+require_relative '../helpers/session_helper'
+
+class ApplicationController < Sinatra::Base
+  register Sinatra::Session
+  register Sinatra::ActiveRecordExtension
+  helpers UserHelper
+  helpers SessionHelper
+
+  Dotenv.load
+  SESSION_SECRET = ENV['SESSION_SECRET']
```

Comment:
what is the reason to set constant? we use it only once during configuration, just put it as is

Reply:
And where is .env.sample?